cypress/support/step_definition/course_management/courseManagementSteps.spec.js

```diff
@@ -0,0 +1,40 @@
+import { Then, When } from "@badeball/cypress-cucumber-preprocessor";
+import CourseManagement from "../../pages/course_management/pageCourseManagement";
+
+const courseManagement = new CourseManagement();
+
+Then("I see the icon Lock in the course {string}", (courseName) => {
+	courseManagement.seeLockIconInCourse(courseName);
+});
+
+Then("I see a message that the course is not accessible", () => {
+	courseManagement.seeCourseNotAccessibleMessage();
+});
+
+When("I click on the locked course {string}", (courseName) => {
+	courseManagement.clickLockedCourse(courseName);
+});
+
+Then("I see two tabs Current and Archive", () => {
```

Comment:
since we already have this method in other file so creating another similar method does not make any sense. you can rename that step instead